Jenkinsfile

```diff
@@ -16,9 +16,48 @@ stage("Unit Test") {
         export LD_LIBRARY_PATH=/usr/local/cuda-10.1/lib64
         export MPLBACKEND=Agg
         export MXNET_CUDNN_AUTOTUNE_DEFAULT=0
-        python3 -m pip install --upgrade --force-reinstall -e .
-        python3 -m pip install pytest
+
+        pip uninstall -y autogluon
+        pip uninstall -y autogluon.vision
+        pip uninstall -y autogluon.text
+        pip uninstall -y autogluon.mxnet
+        pip uninstall -y autogluon.extra
+        pip uninstall -y autogluon.tabular
+        pip uninstall -y autogluon.core
+
+        cd core/
```

Comment:
one issue I had run into when benchmarking was I needed to upgrade setuptools to work with namespace packages.

```pip install -U setuptools```

This had to be done prior to any installs.

Perhaps we add this to install FAQ?

Reply:
added to readme and FAQ

Reply:
added link to the old docs 
![image](https://user-images.githubusercontent.com/10080307/95641065-fd694f80-0a54-11eb-9aa6-8a67861cd4b7.png)